api/config/v1alpha1/helpers.go

```diff
@@ -99,12 +100,18 @@ func DefaultKubernetesDeploymentReplicas() *int32 {
 	return &repl
 }
 
+// DefaultKubernetesDeploymentImage returns the default envoyproxy image.
+func DefaultKubernetesDeploymentImage() *string {
+	return pointer.String(DefaultEnvoyProxyImage)
+}
+
 // DefaultKubernetesDeployment returns a new KubernetesDeploymentSpec with default settings.
 func DefaultKubernetesDeployment() *KubernetesDeploymentSpec {
 	return &KubernetesDeploymentSpec{
 		Replicas:  DefaultKubernetesDeploymentReplicas(),
 		Pod:       DefaultKubernetesPod(),
 		Container: DefaultKubernetesContainer(),
+		Image:     DefaultKubernetesDeploymentImage(),
```

Comment:
since the API has now changed, should `Image` live under `Container` ?

Reply:
Done